backend/Tasque.BLL/Services/UserService.cs

```diff
@@ -42,6 +42,20 @@ public async Task<UserDto> GetUserById(int id)
             return userDto;
         }
 
+        public async Task<UserDto> GetUserByEmail(string email)
+        {
+            var userEntity = await _context.Users.FirstOrDefaultAsync(u => u.Email == email)
```

Comment:
Maybe it's worth checking for EmailConfirmed as well ?

Reply:
Added checking for EmailConfirmed.